pom.xml

```diff
@@ -201,7 +189,7 @@
               <goal>copy-resources</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/${project.artifactId}/js</outputDirectory>
+              <outputDirectory>${project.basedir}/src/main/webapp/js</outputDirectory>
```

Comment:
https://github.com/jenkinsci/prism-api-plugin/pull/41/files#r893895710

Reply:
also note that without this change it is unusable in `mvn hpi:run`.
Yes this could be better in `maven-hpi-plugin` but better the plugin works in development mode than being completely broken